### PR TITLE
feat: Access Risk support ETH chain

### PR DESCRIPTION
**Pull request by ChefMomota**
---

### Detailed summary
- Added `ACCESS_TOKEN_SUPPORT_CHAIN_IDS` constant to `supportedChains.ts` file in `components/AccessRisk/config` directory.
- Imported and used `ACCESS_TOKEN_SUPPORT_CHAIN_IDS` constant in various files to determine if the current chain is supported.
- Updated conditional rendering and logic based on the value of `ACCESS_TOKEN_SUPPORT_CHAIN_IDS`.
- Removed `SUPPORT_ONLY_BSC` constant from `config/constants/supportChains.ts` file.
- Added `useActiveChainId` hook import to `AccessRiskTooltips.tsx` file in `components/AccessRisk` directory.
- Updated conditional rendering and logic in `AccessRiskTooltips.tsx` file based on the value of `chainId`.
- Imported and used `ACCESS_TOKEN_SUPPORT_CHAIN_IDS` constant in `SwapFeaturesContext.tsx` file in `views/Swap` directory.
- Updated conditional rendering and logic in `SwapFeaturesContext.tsx` file based on the value of `ACCESS_TOKEN_SUPPORT_CHAIN_IDS`.
- Imported and used `ACCESS_TOKEN_SUPPORT_CHAIN_IDS` constant in `index.tsx` file in `views/LimitOrders` directory.
- Updated conditional rendering and logic in `index.tsx` file based on the value of `ACCESS_TOKEN_SUPPORT_CHAIN_IDS`.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: apps/web/src/components/AccessRisk/index.tsx [hunk 1]
```diff
@@ -11,17 +10,18 @@ import {
...
+import { fetchRiskToken } from 'components/AccessRisk/utils/fetchTokenRisk'
 import { useEffect, useMemo, useState } from 'react'
-import { useUserTokenRisk } from 'state/user/hooks/useUserTokenRisk'
 import { useAllLists } from 'state/lists/hooks'
+import { useUserTokenRisk } from 'state/user/hooks/useUserTokenRisk'
 import { styled } from 'styled-components'
 import useSWRImmutable from 'swr/immutable'
-import { fetchRiskToken } from 'components/AccessRisk/utils/fetchTokenRisk'
-import AccessRiskTooltips from 'components/AccessRisk/AccessRiskTooltips'
+import { ACCESS_TOKEN_SUPPORT_CHAIN_IDS } from 'views/Swap/SwapFeaturesContext'
```
Comment (chefjackson) [inline apps/web/src/components/AccessRisk/index.tsx:24]: Let's move this constant out from Swap views if it's used somewhere else. Let's put it under `src/constants`

Comment (chefjackson) [reply apps/web/src/components/AccessRisk/index.tsx:24]: maybe add to `src/config/constants/supportedChains`

File: apps/web/src/components/AccessRisk/AccessRiskTooltips.tsx [hunk 2]
```diff
@@ -47,12 +50,14 @@ const AccessRiskTooltips: React.FC<AccessRiskTooltipsProps> = ({
...
-        <Flex mt="4px">
-          <Text>{t('Get more details from')}</Text>
-          <Link ml="4px" external href={`https://dappbay.bnbchain.org/risk-scanner/${tokenAddress}`}>
-            {t('RedAlarm')}
-          </Link>
-        </Flex>
+        {chainId === ChainId.BSC && (
+          <Flex mt="4px">
+            <Text>{t('Get more details from')}</Text>
+            <Link ml="4px" external href={`https://dappbay.bnbchain.org/risk-scanner/${tokenAddress}`}>
```
Comment (ChefJerry) [inline apps/web/src/components/AccessRisk/AccessRiskTooltips.tsx:56]: can not see risk info from this url: https://dappbay.bnbchain.org/risk-scanner/0xa80221D067603e30060F75E2458AA361f8eE5402

not sure if is network problem

Comment (chefjackson) [reply apps/web/src/components/AccessRisk/AccessRiskTooltips.tsx:56]: looks ok on my end

Comment (ChefMomota) [reply apps/web/src/components/AccessRisk/AccessRiskTooltips.tsx:56]: same looks ok on my end

File: apps/web/src/components/AccessRisk/index.tsx (hunk 1 above)
Comment (ChefMomota) [reply apps/web/src/components/AccessRisk/index.tsx]: Done. Move under `AccessRisk`

**Review by chefjackson — APPROVED** [review]

**Pull request merged**